test/integration/timeout.spec.js

```diff
@@ -18,4 +18,73 @@ describe('this.timeout()', function () {
       done();
     });
   });
+
+  describe('0 (no timeout)', function () {
+    var itIfNodeWorthy = (function () {
```

Comment:
happily, we can ditch this.

(I *am* merging and fiddling with this code, so I don't expect you to do it)